ui/StatusQ/src/StatusQ/Popups/StatusMenu.qml

```diff
@@ -88,6 +88,11 @@ Menu {
     delegate: StatusMenuItem {
         visible: root.hideDisabledItems ? enabled : true
         height: visible ? implicitHeight : 0
+        onImplicitWidthChanged: {
+            var adjustedImplicitWidth = implicitWidth + 2 * padding
```

Comment:
```suggestion
            const adjustedImplicitWidth = implicitWidth + leftPadding + rightPadding
```

Reply:
There is no need to add `StatusMenuItem`'s padding here because it's already included in the `implicitWidth` of that component.
So it can be just:
```
root.contentWidth = Math.max(root.contentWidth, implicitWidth)
```
![Screenshot from 2023-06-15 15-51-05](https://github.com/status-im/status-desktop/assets/20650004/c2dc4bfe-6ac3-47d4-9651-2418cb273d65)